### PR TITLE
Add some missing cfi frame info in aesni-x86_64.pl

### PR DESCRIPTION
**Pull request by bernd-edlinger**
##### Checklist

- [ ] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

File: crypto/aes/asm/aesni-x86_64.pl [hunk 1]
```diff
@@ -4710,6 +4725,7 @@ sub aesni_generate8 {
 	shufps	\$0b10101010,%xmm1,%xmm1	# critical path
 	xorps	%xmm1,%xmm2
 	ret
+.cfi_endproc
```
Comment (kroeckx) [inline crypto/aes/asm/aesni-x86_64.pl]: Lkey_expansion_256a and key_expansion_256b are something that's called, so maybe they should should have thrown own .cfi_startproc and .cfi_endproc

Comment (kroeckx) [reply crypto/aes/asm/aesni-x86_64.pl]: Oh, for added fun, key_expansion_256a_cold also has it's own one.

File: crypto/aes/asm/aesni-x86_64.pl [hunk 2]
```diff
@@ -986,6 +986,7 @@ sub aesni_generate8 {
 .type	aesni_ccm64_encrypt_blocks,\@function,6
 .align	16
 aesni_ccm64_encrypt_blocks:
+.cfi_startproc
 ___
 $code.=<<___ if ($win64);
 	lea	-0x58(%rsp),%rsp
```
Comment (kroeckx) [inline crypto/aes/asm/aesni-x86_64.pl]: Should we do something with the CFA here?

Comment (bernd-edlinger) [reply crypto/aes/asm/aesni-x86_64.pl]: no, I can't do anything for win64

Comment (bernd-edlinger) [reply crypto/aes/asm/aesni-x86_64.pl]: win64 unwind info is totally broken, I wonder if it would
be better to remove that altogether, @davidben ?

Comment (kroeckx) [reply crypto/aes/asm/aesni-x86_64.pl]: I missed the win64 part, but saw the win64 part in one of the other PRs.

Comment (bernd-edlinger) [reply crypto/aes/asm/aesni-x86_64.pl]: as far as I can tell the .cfi directives are ignored by windows.
The unwinding cannot not work correctly when this block is interrupted.
It appears to be correcting the stack and restoring xmm6-9 only after
.Lccm64_enc_body:
but when interrupted directly after the lea only the stack needs to be corrected,
but the registers are not yet saved.
I wonder how that stuff was tested.

**Comment by bernd-edlinger** [comment]
Merge to master and 1.1.1
Thanks!

**Pull request closed without merging**